build.gradle

```diff
@@ -980,7 +980,7 @@ project(':streams') {
 project(':streams:streams-scala') {
   println "Building project 'streams-scala' with Scala version ${versions.scala}"
   apply plugin: 'scala'
-  archivesBaseName = "kafka-streams-scala"
+  archivesBaseName = "kafka-streams-scala_${versions.baseScala}"
```

Comment:
Unrelated question, why is `scalaLogging` just a test compile dependency?

Reply:
It's required by the `LazyLogging` trait referenced by several tests, but it doesn't look like any logging is actually done.  It's probably safe to remove this.  @debasishg can you provide some insight here?

Reply:
+1 .. it's there, along with `log4j.properties` as part of logging infrastructure from the tests. Currently not used though. We can remove the imports and do away with the dependency. Do we keep `log4j.properties` ?

Reply:
Let's remove the dependency.  I think it makes sense to keep `log4j.properties` so users can see what's happening with the embedded Kafka instance and Kafka streams when the tests are run.